### ISSUE TITLE
use forward-backward posterior probabilities for painting

### ISSUE THREAD

**Issue by shajoezhu** [mcveanlab/DEploid#172]
#170

**Comment by shajoezhu** [comment]
working progress
```bash
./dEploid_dbg -vcf data/exampleData/PG0390-C.eg.vcf -plaf data/exampleData/labStrains.eg.PLAF.txt -o PG0390-CPanelExclude -panel data/exampleData/labStrains.eg.panel.txt -painting PG0390-CPanel.hap -initialP 0.8 0.2 -k 2
less PG0390-CPanelExclude.single1
less PG0390-CPanelExclude.single0
```

**Comment by shajoezhu** [comment]
closed by #170
